apps/i18n/studio/en_us.json

```diff
@@ -483,5 +483,12 @@
   "whenTouchWallTooltip": "Execute the actions below when the actor touches an obstacle.",
   "whenUp": "when up arrow",
   "whenUpTooltip": "Execute the actions below when the up arrow key is pressed.",
-  "yes": "Yes"
+  "yes": "Yes",
+  "failedHasSetSprite": "Next time, set a character.",
+  "failedHasSetBotSpeed": "Next time, set a bot speed.",
+  "failedTouchAllItems": "Next time, get all the items.",
+  "failedScoreMinimum": "Next time, reach the minimum score.",
+  "failedRemovedItemCount": "Next time, get the right number of items.",
+  "failedSetActivity": "Next time, set the correct character activity."
```

Comment:
It probably makes sense to parameterise these a little so they can give explicit counts & values, etc.?